_includes/sxs/02-extending/part-three.html

```diff
@@ -93,12 +93,12 @@ <h3 class="step">Add stylesheet partial.</h3>
     <h4 class="filename"><span class="material-icons" aria-hidden="true">description</span> _theme/_uswds-theme-custom-styles.scss</h4>
 <pre class="blur">@use "uswds-core" as *;
 @forward "../assets/css/styles.scss";
-@forward "../assets/css/_my-card.scss"; 
-</pre><pre class="add">@forward "../assets/css/_my-testimonial.scss";</pre>
+@forward "../assets/css/_my-card.scss";
+</pre><pre class="add">@forward "../assets/css/components/_my-testimonial.scss";</pre>
```

Comment:
We changed the instructions to create the components dir in `_theme` 

```suggestion
@forward "../_theme/components/_my-card.scss";
</pre><pre class="add">@forward "../_theme/components/_my-testimonial.scss";</pre>
```